src/output/outputMod.F90

```diff
@@ -1499,7 +1507,25 @@ subroutine storeSurfsolInBuffer(sps, buffer, nn, blockID,   &
        rangeFace(2,1:2) = rangeCell(3,1:2)
        iiMax = jl; jjMax = kl
 
-       ww1    => w(1,1:,1:,:);   ww2    => w(2,1:,1:,:)
+       ! mham have to get the mesh coordinates further down in order to compute
+       ! the correct Cp-normalisation for rotational setups with Wind Turbines.
+       ! Careful now; the flow variables, w(0:ib,0:jb,0:kb,1:nw), we point to
+       ! below uses what is defined in l.216-217, block.F90 as
```

Comment:
I guess we can refer to `type blockType` in `block.F90` here. Anything to add/remove about this comment block?

Reply:
why do we need to access these from the `w` array? Can we get this info from BC face info?

Reply:
@SichengHe :eyes: